runtime/default/build.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
+# This script is run inside a Lambda container to compile PHP for Lambda's OS.
+# It will export a ZIP usable as a Lambda Layer.
+# See https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-path
+#
+# You must specify the PHP version using the `PHP_VERSION` environment variable.
+
+set -e
+
+if [[ -z "$PHP_VERSION" ]] ; then
+    echo 'The PHP version to build must be passed in the PHP_VERSION environment variable'
+    exit 1
+fi
+
+# Install dependencies
+# --releasever=2017.03: Lambda is based on 2017.03, dont' grab the latest revisions of development packages.
+# findutils: because the pecl command uses find
+# libicu and libicu-devel: because C++ & ICU are required by INTL
+yum -y --releasever=2017.03 install \
+    autoconf \
+    automake \
+    libtool \
+    bison \
+    re2c \
+    libxml2-devel \
+    openssl-devel \
+    libpng-devel \
+    libjpeg-devel \
+    curl-devel \
+    findutils \
+    php-pear \
+    libicu \
+    libicu-devel \
+    c++ \
+    gcc-c++
+
+# Compile PHP
+mkdir /tmp/bref
+cd /tmp/bref
+curl -sL https://github.com/php/php-src/archive/php-$PHP_VERSION.tar.gz | tar -zxv
```

Comment:
wget fallback when curl is not available?

Reply:
Do you have a replacement that works for that line?